<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>String Matcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Mathcer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item row="2" column="4">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;18thCentury&quot;;</string>
            </property>
            <property name="text">
             <string>Prefix / Suffix Width</string>
            </property>
           </widget>
          </item>
          <item row="2" column="7">
           <widget class="QPushButton" name="bt_reset">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>   Reset   </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;18thCentury&quot;;</string>
            </property>
            <property name="text">
             <string>Result:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QPushButton" name="bt_solve">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>   Match   </string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="8">
           <widget class="QTextEdit" name="t_output">
            <property name="styleSheet">
             <string notr="true">font: 33pt &quot;微软雅黑&quot;;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="8">
           <widget class="QTextEdit" name="t_input">
            <property name="styleSheet">
             <string notr="true">font: 33pt &quot;微软雅黑&quot;;</string>
            </property>
            <property name="placeholderText">
             <string>Input...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="5">
           <widget class="QSpinBox" name="s_width">
            <property name="styleSheet">
             <string notr="true">font: 22pt &quot;18thCentury&quot;;</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2" rowspan="2">
           <widget class="QComboBox" name="comboBox">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
            </property>
            <item>
             <property name="text">
              <string>正常</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>优雅暗橙</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>清新白蓝</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>典雅炫黑</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="status_1">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;18thCentury&quot;;</string>
          </property>
          <property name="text">
           <string>Ready.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>History</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="2">
           <widget class="QTextEdit" name="hout">
            <property name="styleSheet">
             <string notr="true">font: 33pt &quot;微软雅黑&quot;;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;18thCentury&quot;;</string>
            </property>
            <property name="text">
             <string>Output</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;18thCentury&quot;;</string>
            </property>
            <property name="text">
             <string>List</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QTextEdit" name="hin">
            <property name="styleSheet">
             <string notr="true">font: 33pt &quot;微软雅黑&quot;;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;18thCentury&quot;;</string>
            </property>
            <property name="text">
             <string>Input</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="4">
           <widget class="QListWidget" name="hlist">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 33pt &quot;微软雅黑&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="status_2">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;18thCentury&quot;;</string>
          </property>
          <property name="text">
           <string>No record yet.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
